src/strategies/stakedotlink-vesting/index.ts

```diff
@@ -0,0 +1,42 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+
+export const author = 'stakedotlink';
+export const version = '0.0.1';
+
+const abi = [
+  'function balanceOf(address _account) public view returns (uint256)',
+  'function totalBalanceOf(address _account) public view returns (uint256)'
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+): Promise<Record<string, number>> {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  const multi = new Multicaller(network, provider, abi, { blockTag });
+  addresses.forEach((address) =>
+    multi.call(address, options.address, 'balanceOf', [address])
+  );
+  const balanceResult: Record<string, BigNumberish> = await multi.execute();
+  addresses.forEach((address) =>
+    multi.call(address, options.address, 'totalBalanceOf', [address])
+  );
+  const totalBalanceResult: Record<string, BigNumberish> =
+    await multi.execute();
+
+  return Object.fromEntries(
+    Object.entries(balanceResult).map(([address, balance]) => {
+      const vestingBalance =
+        parseFloat(formatUnits(totalBalanceResult[address], options.decimals)) -
+        parseFloat(formatUnits(balance, options.decimals));
```

Comment:
Hey @jleeh Why is it minus? Can add a README for everyone to understand? TY!

Reply:
Hey @ChaituVR, added more context to the readme. `totalBalanceOf` returns an accounts balance plus the amount of tokens currently vesting, whereas `balanceOf` is just the amount of tokens an account has. `totalBalanceOf - balanceOf` then returns the amount currently vesting.

This strategy would be used in conjunction with a standard ERC20 balanceOf strategy to calculate total voting weight.